src/client/dfuse/il/int_posix.c

```diff
@@ -807,11 +807,6 @@ child_hdlr(void)
 		DL_WARN(rc, "daos_eq_lib_init() failed in child process");
 	daos_dti_reset();
 	ioil_eqh = ioil_iog.iog_main_eqh = DAOS_HDL_INVAL;
```

Comment:
i don't think we should do this for IL.
IL only intercepts read/write calls unlike pil4dfs. So it's more likely that that context is going to be used in this case.

Reply:
child_hdlr() is executed once a child process is forked if the parent process calls open() to access files on daos. We do not know whether the child process only calls metadata related functions or not.

Reply:
this was not my point.
IOIL does not intercept any metadata or libaio calls, so creating a context here is not going to matter much and is only beneficial in case the child actually does IO that is intercepted.
But in pil4dfs case, the child will create additional network context now if it does libaio for example, and we already started creating aio context here taking away a resource from there.

Reply:
ok. Just reverted the change in libioil.